### ISSUE TITLE
Add support for `its` similar to rspec

### ISSUE THREAD

**Issue by stalniy** [stalniy/bdd-lazy-var#25]
- [x] `its` implementation
- [x] `it` without description implementation

```js
describe("User", () => {
  subject() => new User())

  its("role", () => expected.to.equal("member"))
})
```

Basically `its` is a wrapper around `it` and example above can be converted to

```js
it("s role equals to member", () => {
  expect($subject.role).to.equal("member")
})
```

**Comment by stalniy** [comment]
Syntax like this will not require to add one more global variable:

```js
its('role', is => is.expected.to.equal("member"))
```

**Comment by stalniy** [comment]
@dkniffin I think it would be a good addition!

**Comment by stalniy** [comment]
available in `bdd-lazy-var@2.4.0`
